provider/source/src/normalizer/mod.rs

```diff
@@ -26,7 +27,11 @@ macro_rules! normalization_provider {
                 let $toml_data: &normalizer_serde::$serde_struct =
                     self.icuexport()?.read_and_parse_toml(&format!(
                         "norm/{}/{}.toml",
-                        self.trie_type(),
+                        if $file_name == "nfd" || $file_name == "nfkd" {
+                            TrieType::Fast
+                        } else {
+                            self.trie_type()
```

Comment:
suggestion: document that the `trie_type` does not apply to these. 

https://unicode-org.github.io/icu4x/rustdoc/icu_provider_source/struct.SourceDataProvider.html#method.with_fast_tries

Reply:
Documented.